source/declarations.tex

```diff
@@ -2419,6 +2432,12 @@
   (int(a))+1;                   // expression
   (int(unsigned(a)))+1;         // type-id (ill-formed)
 }
+
+typedef struct BB { int C[2]; } *B, C;
+void g() {
+  sizeof(B()->C[1]);            // OK, \tcode{sizeof(}expression\tcode{)}
+  sizeof(auto()->C[1]);         // error: \tcode{sizeof} of a function returning an array
```

Comment:
See "expression" in comments above -` \grammarterm` is not used. It's not clear whether we literally mean the `\grammarterm` expression here.

Applied suggested use of `\keyword`, tho we don't tend to mark keywords in the comments of examples. Do we want to?

Reply:
We're inconsistent with keyword markup. I think not doing it in example is fine; it clutters the index.

Yes, we mean the grammarterm here: The comment says that the indicated code is parsed as `sizeof(_expression_)`. This is all about parsing, not about any further expression analysis (e.g. value category etc.)

Reply:
Then, since /expression/ wasn't marked elsewhere in this example, I think it's better to not mark it here (else one would assume the others aren't grammarterms). Leaving as is.